src/main/java/com/sap/oss/phosphor/fosstars/model/value/VersionRange.java

```diff
@@ -0,0 +1,67 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+/**
+ * The VersionRange holds start and end version of a CVE.
```

Comment:
It doesn't seem to be connected to CVE/CPE.

```suggestion
 * The class holds start and end version.
```